bedrock/settings/base.py

```diff
@@ -1181,6 +1184,7 @@ def get_default_gateway_linux():
     "accounts.firefox.com",
     "accounts.firefox.com.cn",
     "www.youtube.com",
+    "cdn.cookielaw.org",
```

Comment:
I'm also not sure we need this. Is there an iframe somewhere?

Reply:
No i think this is good to remove! good catch